web/index.html

```diff
@@ -35,7 +35,11 @@
 		</select>
 		<p style="display:inline" id="basic-stats"></p>
 
-		<canvas id="chart"></canvas>
+		<!-- max-width is ~2:1 (chart aspect ratio) to height, so canvas doesn't get too tall -->
+		<div style="max-width: 190vh; margin: 0 auto;">
```

Comment:
What is the rationale for going with `190` and not `200`?

Reply:
200 would fill up the entire screen with the chart. I made it smaller so we could see the other interface elements as well